plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -67,11 +68,23 @@ export class PgInstrumentation extends InstrumentationBase {
         if (isWrapped(moduleExports.Client.prototype.query)) {
           this._unwrap(moduleExports.Client.prototype, 'query');
         }
+
+        if (isWrapped(moduleExports.Client.prototype.connect)) {
+          this._unwrap(moduleExports.Client.prototype, 'connect');
+        }
+
         this._wrap(
           moduleExports.Client.prototype,
           'query',
           this._getClientQueryPatch() as never
         );
+
+        this._wrap(
+          moduleExports.Client.prototype,
+          'connect',
+          this._getClientConnectPatch() as never
```

Comment:
Using `as never` to avoid type checker complaining is silly. Fine if you don't want to bloat the PR fixing an unrelated issue, but we shouldn't add more.

Reply:
This instrumentation seems to use `as never` everywhere in the code, I changed them to `as any` now like all other instrumentations. The connect function itself is overloaded.